schedule/src/main/java/itcast/news/application/NewsService.java

```diff
@@ -85,7 +81,7 @@ public List<String> findLinks(String url) throws IOException {
         return links;
     }
 
-    List<String> isValidLinks(List<String> links) {
+    public List<String> isValidLinks(List<String> links) {
         List<String> isValidLinks = newsRepository.findAllLinks();
         return links
                 .stream()
```

Comment:
이거는 그냥 띄어쓰기 하실 때 
```suggestion
        return links.stream()
```
이렇게 적어주셔도 좋을거 같습니다!! .stream() 앞에서 줄바꿈하지 않아도 스트림 변환한거 눈에 보여서요!